src/server/scripts/Outland/BlackTemple/black_temple.cpp

```diff
@@ -29,198 +29,262 @@ Comment:  Spirit of Olum: Player Teleporter to Seer Kanai Teleport after defeati
 
 enum Spells
 {
-    // Spirit of Olum
-    SPELL_TELEPORT                   = 41566,
-    // Wrathbone Flayer
-    SPELL_CLEAVE                     = 15496,
-    SPELL_IGNORED                    = 39544,
-    SPELL_SUMMON_CHANNEL             = 40094
+	// Spirit of Olum
+	SPELL_TELEPORT = 41566,
+	SPELL_TELEPORT_2 = 41570,
+	// Wrathbone Flayer
+	SPELL_CLEAVE = 15496,
+	SPELL_IGNORED = 39544,
+	SPELL_SUMMON_CHANNEL = 40094
 };
 
 enum Creatures
 {
-    NPC_BLOOD_MAGE                   = 22945,
-    NPC_DEATHSHAPER                  = 22882
+	NPC_BLOOD_MAGE = 22945,
+	NPC_DEATHSHAPER = 22882
 };
 
 enum Events
 {
-    // Wrathbone Flayer
-    EVENT_GET_CHANNELERS             = 1,
-    EVENT_SET_CHANNELERS             = 2,
-    EVENT_CLEAVE                     = 3,
-    EVENT_IGNORED                    = 4,
+	// Wrathbone Flayer
+	EVENT_GET_CHANNELERS = 1,
+	EVENT_SET_CHANNELERS = 2,
+	EVENT_CLEAVE = 3,
+	EVENT_IGNORED = 4,
 };
 
 // ########################################################
 // Spirit of Olum
 // ########################################################
+#define OZ_GOSSIP1          "Take me to the other Deathsworn, Olum."
```

Comment:
@DDuarte I understand, I will be in accordance with the new standards and try again the next.